### PR TITLE
User can view a single level.

### PR DESCRIPTION
**Pull request by gitrkaplan**
User is now able to view a single level. All markup done in javaScript and styled accordingly in css/styles.css

### CODE REVIEW

**Comment by gitrkaplan** [comment]
screen shot 2017-09-13 at 8 19 31 pm
screen shot 2017-09-13 at 8 20 06 pm

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
+// function to take in entire round and returns a $round DOM tree
```
Comment (thebearingedge) [inline js/main.js]: You can delete explanatory comments like this. Rely instead on great names for your variables and functions.

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  var $roundNumber = document.createElement('h4')
+  $roundNumber.setAttribute('id', 'round-number')
+
+  var $span = document.createElement('span')
+  $span.setAttribute('id', 'current-round')
+  $span.textContent = round.roundNumber
+
+  $roundNumber.textContent = 'Round: ' + '$span'
+
+  // 2 children including text node and span
```
Comment (thebearingedge) [inline js/main.js]: What's this comment for?

**Comment by thebearingedge** [comment]
Please reopen when ready.

File: js/main.js (hunk 2 above)
Comment (gitrkaplan) [reply js/main.js]: Whoops, i'll remove it. It was a note from when we were discussing this.

**Comment by gitrkaplan** [comment]
Fixed issues related to markup. Span element is now dynamically appended and comments removed.

**Pull request merged**